cmds/ed/command.go

```diff
@@ -41,7 +42,8 @@ func Command(f Editor, c string, startLine, endLine int) error {
 	case 'r':
 		fname := strings.TrimLeft(a, " \t")
 		debug("read %v @ %v, %v", f, startLine, endLine)
-		r, err := os.Open(fname)
```

Comment:
It is a bit subtle. This `:=` defines a new `err` which shadows the original definition on line 16.